.idea/libraries/org_antlr_antlr4_4_5_1_1.xml

```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```

Comment:
You can remove it, by itself it does not help to automate the dependencies -- it is not a maven input file AFAIK. 

I write the following just so that you can follow my line of though and see if it is the same of yours:
I guess it should be necessary to have some maven file with such dependencies and run maven if necessary when you run the tests, which implies you should have maven on your test server.